src/zenml/integrations/feast/__init__.py

```diff
@@ -0,0 +1,36 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""
+The AWS integration submodule provides a way to run ZenML pipelines in a cloud
+environment. Specifically, it allows the use of cloud artifact stores,
+and an `io` module to handle file operations on S3 buckets.
+"""
+
+from zenml.integrations.constants import FEAST
+from zenml.integrations.integration import Integration
+
+
+class FeastIntegration(Integration):
+    """Definition of Feast integration for ZenML."""
+
+    NAME = FEAST
+    REQUIREMENTS = ["feast[redis]", "redis-server"]
+
+    @classmethod
+    def activate(cls) -> None:
+        """Activates the integration."""
+        from zenml.integrations.feast import feature_stores  # noqa
```

Comment:
Perhaps this file should be called `feature_store` not `feature_stores`

Reply:
That would go against the pattern that we use everywhere else in our codebase. What would be the reasoning for changing the name?

Reply:
Actually you are right, disregard my comment